README.md

```diff
@@ -1,4 +1,4 @@
-[![Tests](https://github.com/ufs-community/uwtools/actions/workflows/test.yaml/badge.svg)](https://github.com/ufs-community/uwtools/actions) [![Release](https://github.com/ufs-community/uwtools/actions/workflows/release.yaml/badge.svg)](https://github.com/ufs-community/uwtools/releases) [![Docs](https://readthedocs.org/projects/uwtools/badge/?version=main)](https://uwtools.readthedocs.io/en/main/?badge=main) [![pkgpage](https://anaconda.org/ufs-community/uwtools/badges/version.svg)](https://anaconda.org/ufs-community/uwtools) [![pkgfiles](https://anaconda.org/ufs-community/uwtools/badges/latest_release_date.svg)](https://anaconda.org/ufs-community/uwtools/files) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/ufs-community/uwtools/notebooks?labpath=notebooks%2Fexample.ipynb)
+[![Tests](https://github.com/ufs-community/uwtools/actions/workflows/test.yaml/badge.svg)](https://github.com/ufs-community/uwtools/actions) [![Release](https://github.com/ufs-community/uwtools/actions/workflows/release.yaml/badge.svg)](https://github.com/ufs-community/uwtools/releases) [![Docs](https://readthedocs.org/projects/uwtools/badge/?version=main)](https://uwtools.readthedocs.io/en/main/?badge=main) [![pkgpage](https://anaconda.org/ufs-community/uwtools/badges/version.svg)](https://anaconda.org/ufs-community/uwtools) [![pkgfiles](https://anaconda.org/ufs-community/uwtools/badges/latest_release_date.svg)](https://anaconda.org/ufs-community/uwtools/files)
```

Comment:
Oops. I forgot to remove the Binder button here and the link in the main page of the user guide when I removed the example notebook.